test/cql-pytest/test_materialized_view.py

```diff
@@ -808,3 +808,14 @@ def test_mv_with_only_primary_key_rows(scylla_only, cql, test_keyspace):
             nodetool.flush(cql, view)
             assert(set([row.id for row in cql.execute(f'SELECT id FROM {view}')]) == set([1, 2, 3]))
             # We now believe that empty value serialization/deserialization is correct
+
+def test_mv_prepared_statement_with_altered_base(scylla_only, cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'id int PRIMARY KEY, v1 int') as base:
```

Comment:
Does this test fail before the fix? If so it should be marked xfail (and the xfail removed in the next patch), so it can be bisected.

Reply:
Sorry, I will revert the order and also remove the scylla_only flag as @nyh requested in the previous cycle.